common/display/src/main/scala/uk/ac/wellcome/display/models/ImagesIncludes.scala

```diff
@@ -0,0 +1,22 @@
+package uk.ac.wellcome.display.models
+
+sealed trait ImageInclude
+
+object ImageInclude {
+  case object VisuallySimilar extends ImageInclude
+}
+
+case class SingleImageIncludes(includes: List[ImageInclude]) {
+  def visuallySimilar = includes.contains(ImageInclude.VisuallySimilar)
+}
+
+object SingleImageIncludes {
+  import ImageInclude._
+
+  def apply(visuallySimilar: Boolean = false): SingleImageIncludes =
+    SingleImageIncludes(
+      List(
+        if (visuallySimilar) Some(VisuallySimilar) else None
+      ).flatten
```

Comment:
Can do `option.toList` instead of `List(option).flatten` here

Reply:
Yeah, going to keep as is though because I anticipate more includes will be added and so want it to match https://github.com/wellcomecollection/catalogue/blob/master/common/display/src/main/scala/uk/ac/wellcome/display/models/WorksIncludes.scala#L42